test/e2e/proxy_test.go

```diff
@@ -104,36 +104,19 @@ func probeClientIPFromNode(node string, baseUrl string, data *TestData) (string,
 }
 
 func probeFromPod(data *TestData, pod, container string, url string) error {
-	var err error
-	if container == busyboxContainerName {
-		_, _, err = data.runWgetCommandOnBusyboxWithRetry(pod, data.testNamespace, url, 5)
-	} else {
-		_, _, err = data.RunCommandFromPod(data.testNamespace, pod, container, []string{"wget", "-O", "-", url, "-T", "5"})
-	}
+	_, _, err := data.runWgetCommandFromTestPodWithRetry(pod, data.testNamespace, container, url, 5)
 	return err
 }
 
 func probeHostnameFromPod(data *TestData, pod, container string, baseUrl string) (string, error) {
 	url := fmt.Sprintf("%s/%s", baseUrl, "hostname")
-	var err error
-	var hostname string
-	if container == busyboxContainerName {
-		hostname, _, err = data.runWgetCommandOnBusyboxWithRetry(pod, data.testNamespace, url, 5)
-	} else {
-		hostname, _, err = data.RunCommandFromPod(data.testNamespace, pod, container, []string{"wget", "-O", "-", url, "-T", "5"})
```

Comment:
The timeout to run wget in `RunCommandFromPod` is 5, but it's 1 in `runWgetCommandFromTestPodWithRetry`, any risk to change this?

Reply:
I don't why the timeout to run wget is 5s, and the timeout for curl is 1s. If you want to change it back, I'm okay with that.

Reply:
I am Ok not to change it back if both works, just wonder if this is added with purpose. I checked the history, and seems it was added by you originally https://github.com/antrea-io/antrea/commit/66208c5ea786d002035b90b059f2fab716e56d0a#diff-5b235b7114a754117b4d8e52a9e9f2eb639dfe5a6930a130f2c5fb7826490295, You may double check.

Reply:
I prefer 1s now since retry is added.